formal-verification/FormalVerification.lean

```diff
@@ -1,7 +1,7 @@
 import ProvenZk.Gates
 import ProvenZk.Ext.Vector
 
-namespace Poseidon2
+namespace VerifyProof
```

Comment:
This changing is annoying (will happen every time you swap gadgets) – we should allow injecting this namespace name when exporting the circuit and set it manually in this project, given the incremental nature of this.